services/keyword_service/keyword_test.go

```diff
@@ -101,7 +105,11 @@ func (s *KeywordServiceDbTestSuite) TestGetKeywordResultWithValidKeywordButInval
 	keyword := models.Keyword{UserID: s.userID, Keyword: faker.Name()}
 	db.GetDB().Create(&keyword)
 
-	keywordService := KeywordService{CurrentUserID: 999999}
+	keywordService := KeywordService{
+		CurrentUser: models.User{
+			Model: gorm.Model{ID: 999999},
```

Comment:
This syntax is quite odd. Can you please explain why you went this way to create a User fixture 🤔

Reply:
Because `KeywordService` struct take only `CurrentUserID` which is `uint` type in the first place but right now I decided to take the whole `User` model and the `ID` came from gorm Model struct.

Reply:
But as I change the design, keyword page doesn't need to display email anymore. I will revert back to previous version.

Reply:
Reversed in cc367782840cad65293683843acfc26beb759628